db/structure.sql

```diff
@@ -8,20 +8,6 @@ SET check_function_bodies = false;
 SET client_min_messages = warning;
 SET row_security = off;
 
---
--- Name: plpgsql; Type: EXTENSION; Schema: -; Owner: -
---
-
-CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
-
-
---
--- Name: EXTENSION plpgsql; Type: COMMENT; Schema: -; Owner: -
---
-
-COMMENT ON EXTENSION plpgsql IS 'PL/pgSQL procedural language';
-
-
```

Comment:
There's no reason why your PR should have changed this file.  This is probably an artifact of development.

To fix, run this in your branch:
```
rake db:structure:dump
```

Then if you compare the file against your local `develop` branch (which mirrors `upstream/develop`) you should see no change:
```
git diff develop db/structure.sql
```

Reply:
@patmbolger Does this look right? 

![image](https://user-images.githubusercontent.com/13518870/54146740-f15c7c80-4430-11e9-956f-b2112828b2b3.png)

Reply:
@rmzse no, there should be no differences.  Perhaps your local `develop` branch is out of sync with `upstream` (`upstream == the project Github repo, which you cloned to create `origin` on Github, and then cloned the latter on your local machine).

You could try to compare to `origin` (when in your feature branch):
```
git diff upstream/develop db/structure.sql
```
If there is a difference, just checkout the version from `upstream` and commit that in your feature branch.   Then, when you update your PR, Github will find no difference between your version and `origin`'s version.

Reply:
If you have a slightly different version of Postgresql you may get some of those differences in your `structure.sql` file.  You can see that they are not differences with tables or other db entities that we are defining.
Like @patmbolger said -- then just use the upstream version of the file.

If, in a future PR, you _are_ making changes to some table/db entity, then it's not an issue.  You'd use yours with the db entity changes and whatever else.

Reply:
@patmbolger How can I see that there is a difference? Is it the red colour? I am not used to checking for this kind of stuff so I am not sure what to look for...

@weedySeaDragon I am trying to follow but not entirely sure. I will check for diff against upstream.

Reply:
Is there a diff?

![image](https://user-images.githubusercontent.com/13518870/54308425-484b8880-45ce-11e9-8827-6f4e959fa794.png)

Reply:
Aha ok, I get it! After going into the "Check changes" view here https://github.com/AgileVentures/shf-project/pull/621/files/5ee2b39cfcefa69a61c758664874e7caa98c58ba I see that the red text is smth that has been added. 

And what I think you guys are doing is helping me finding the root cause rather than just deleting those lines. Am I onto smth here? :)